benchmarks/benchmarks/stats.py

```diff
@@ -434,3 +434,16 @@ def time_fni(self, distcase):
             sup.filter(RuntimeWarning, "divide by zero")
             sup.filter(RuntimeWarning, "invalid value encountered")
             stats.NumericalInverseHermite(dist)
+
+
+class DistanceFunctions(Benchmark):
+    def setup(self):
+        rng = np.random.default_rng(12345678)
+        self.u_values, self.u_weights = rng.random(4) * 10, rng.random(4) * 10
+        self.v_values, self.v_weights = rng.random(2) * 10, rng.random(2) * 10
+
+    def time_energy_distance(self):
+        distance = stats.energy_distance(self.u_values, self.v_values, self.u_weights, self.v_weights)
+
+    def time_wasserstein_distance(self):
+        distance = stats.wasserstein_distance(self.u_values, self.v_values, self.u_weights, self.v_weights)
```

Comment:
Could you break this line to make it fit in 80 char line length? And same for the `energy_distance` call above?